code/src/reusable/Urls.js

```diff
@@ -0,0 +1,5 @@
+export const MOVIE_LIST_URL = "https://api.themoviedb.org/3/movie/popular?api_key=54b9b4273ba4db041df9f1cbf5f6875f&language=en-US&page=1";
+
+export const MOVIE_BACKDROP_URL = (backdrop_path) => `https://image.tmdb.org/t/p/w1280${backdrop_path}`;
```

Comment:
Great job creating more URL's but we can see in the moviepage.js that you haven't used this one. You can also create one in the same way for poster_path and then import both to make your code cleaner :)